src/domains/EventDomain.ts

```diff
@@ -50,10 +55,7 @@ class EventDomain implements EventDomainInterface {
    * Summary: undefined
    * Description: Returns a single event by id
    **/
-  public async getEventById(
-    jwtData: JwtAccess,
-    params: EventEventIdGetPath
-  ): Promise<Event> {
+  public async getEventById(jwtData: JwtAccess, params: EventEventIdGetPath): Promise<Event> {
```

Comment:
I think there's a mistake in the specification, this can be retrieved without being authenticated and will the database logic be added in this PR or?

Reply:
Nope there's is no mistake, I just excluded authentication from event router for testing purpose, I will handle that when auth is implemented